bin/verify-exercises-in-docker

keiravillekode opened the pull request:
Adapted from [generic-track](https://github.com/exercism/generic-track/blob/main/bin/verify-exercises-in-docker)

keiravillekode commented:
The following currently fail:

```
bin/verify-exercises-in-docker list-ops
bin/verify-exercises-in-docker robot-name
```

```
Exception: variable srfi-64 is not bound
```

```
Exception: variable build-robot is not bound
```

BNAndras commented:
Any objections to merging this? It looks like @SleeplessByte already thumbs up'ed.

keiravillekode committed:
bin/verify-exercises-in-docker

This script from generic-track is useful for testing exercises
against a test runner image.
